README.md

```diff
@@ -91,6 +91,86 @@ if err != nil {
 log.Print(res) // do something with the result
 ```
 
+## WebSocket Client
+
+Import the WebSocket client and models packages to get started.
+
+```golang
+import (
+    polygonws "github.com/polygon-io/client-go/websocket"
+    "github.com/polygon-io/client-go/websocket/models"
+)
+```
+
+Next, create a new client with your API key and a few other config options.
+
+```golang
+// create a new client
+c, err := polygonws.New(polygonws.Config{
+    APIKey:    "YOUR_API_KEY",
+    Feed:      polygonws.RealTime,
+    Market:    polygonws.Stocks,
+    ParseData: true, // this tells the client to unmarshal data into models
```

Comment:
I think `ParseData` is going to be 99% of clients. Not sure the use case for this, really. But, It would be nice to not require fields ( requiring users time, brainpower ) for majority of use cases.

Reply:
that's a fair point! i could flip it to something like `RawData` so users have to intentionally disable parsing data